addon/templates/components/uni-carrousel.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="uni-carrousel__container">
+    <button class="uni-carrousel__button uni-carrousel__button--prev" {{action "prev"}}>&lt;</button>
+
+    <div class="uni-carrousel__container__wrapper" style="transform: translate3d({{deltaX}}px, 0, 0)">
```

Comment:
Really need this inline style...? If so, put the 'px' in the second 0, for the reasons you stated before